backend/src/main/java/ai/verta/modeldb/dataset/DatasetDAORdbImpl.java

```diff
@@ -617,6 +619,7 @@ public Dataset updateDatasetName(String datasetId, String datasetName)
       // Check entity already exists
       checkDatasetAlreadyExist(dataset);
 
+      Transaction transaction = session.beginTransaction();
```

Comment:
might as well move it two lines below.

Reply:
wouldn't a similar change be required for updateDatasetDescription as well?

Reply:
moved it to all update* functions